inc/util/string_utils.hpp

```diff
@@ -46,6 +46,21 @@ namespace ebi
 
         output.swap(ret);
     }
+
+    /**
+     * Temporal implementation for mismatch with 2 starts and 2 ends. It is not in STL for c++11, but it will in c++14.
```

Comment:
but is the support stable or experimental? I would try to stick with at least the default deployed version with the distros.

Although, now I remember, gcc 4.8 is not c++11-library compliant, so... it invalidates my point. sigh